window.lua

```diff
@@ -0,0 +1,28 @@
+local String = require 'stringutil'
+
+local Window = {}
+
+local WINDOW_FILE = 'window.data'
+
+Window.initialize = function()
+  local fileInfo = love.filesystem.getInfo(WINDOW_FILE)
+  if fileInfo then
+    local contents = love.filesystem.read(WINDOW_FILE)
+    local windowArr = String.split(contents, ',')
+
+    local x = windowArr[1]
+    local y = windowArr[2]
+    local display = windowArr[3]
```

Comment:
oob check? or just fail? not sure if it could fail gracefully if someone did faff with the file.

Reply:
I believe they'd be `nil`, which would probably work with the setWindow function? Like it would use some default instead?